controllers/articleControllers.js

```diff
@@ -0,0 +1,86 @@
+const Article = require('./../models/articleModel');
+
+exports.getAllArticle = async (req, res, next) => {
+  try {
+    const articles = await Article.find();
+    res.status(200).json({
+      status: "Success",
+      Results: articles.length,
+      data: { articles }
+    })
+  } catch (error) {
+    return next(
+      res.status(404).json({
+        status: 'fail',
+        message: "URL NOT FOUND..."
+      })
+    )
+  }
+}
+
+exports.getArticle = async (req, res) => {
+  try {
+    const article = await Article.findById(req.params.id);
+    console.log(req.params.id)
+    res.status(200).json({
+      status: 'success',
+      data: { article }
+    })
+  } catch (error) {
+    res.status(404).json({
+      status: 'fail',
+      message: "URL NOT FOUND"
+    })
+  }
+};
+
+exports.createNewArticle = async (req, res) => {
+  try {
+    const newArticle = await Article.create(req.body);
```

Comment:
How sure are you that `body` contains _exactly_ and _only_ the properties you're looking for?